src/shred_network/shred_receiver.zig

```diff
@@ -32,6 +33,9 @@ const VariantCounter = sig.prometheus.VariantCounter;
 const Logger = sig.trace.Logger("shred_receiver");
 const VerifiedMerkleRoots = sig.utils.lru.LruCache(.non_locking, sig.core.Hash, void);
 
+const DUPLICATE_SHRED_HEADER_SIZE: u64 = 63;
+const DUPLICATE_SHRED_MAX_PAYLOAD_SIZE: u16 = 512;
+
 /// Analogous to [ShredFetchStage](https://github.com/anza-xyz/agave/blob/aa2f078836434965e1a5a03af7f95c6640fe6e1e/core/src/shred_fetch_stage.rs#L34)
 pub const ShredReceiver = struct {
     params: Params,
```

Comment:
Does `Params` need to be a field of `ShredReceiver`? Seems like it would be pretty easy to make this a parameter that's passed to `run`, and then in there to whatever else needs access to these things.

Reply:
I am sure there is a possibility it can be moved, but this was already here and was not added in this PR. I looked into updating it as suggested, but the diff would  be a distraction to the core changes being added in the PR